### ISSUE TITLE
Show ticket wise analytics

### ISSUE THREAD

**Issue by iamareebjamal** [fossasia/open-event-orga-app#182]
Currently, only aggregate analysis of all tickets is done

We need to show ticket wise analysis along with the ticket type

![test](https://user-images.githubusercontent.com/3874064/27297106-295a7500-5541-11e7-98c9-7d55886140c8.png)

**Comment by mejariamol** [comment]
To be done after #185

**Comment by iamareebjamal** [comment]
Taking this up. Let's discuss what should be the approach @mejariamol

**Comment by mejariamol** [comment]
Will need to divide it into sub features.
First feature can be graph of daily ticket sales quantity versus time. (in days or month)
what do you say?

**Comment by iamareebjamal** [comment]
How will we store the time factor? IMO, that is the responsibility of the server. Right now, we should just use what we have. I was thinking to first show the broad ticket analytics of `Free`, `Paid`, and `Donation`

And then in another PR, add the specific Ticket Analytics as to how many of this specific Free ticket was sold

Further into that would be to add feature to see attendees who bought this particular ticket

**Comment by mejariamol** [comment]
>How will we store the time factor?
Can we use order, attendee and ticket to show ticket sales - time graph ?

**Comment by iamareebjamal** [comment]
Hmm, we can. Let's create another issue for that

**Comment by mejariamol** [comment]
:smile: :+1:
